sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,38 +1,37 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
-### Breaking Changes
-
-- `Azure::Core::Http::Url::AppendPath` now does not encode the input by default.
-- Removed `azure.hpp`.
+## 1.0.0-beta.3 (20-11-10)
 
 ### New Features
 - Added `strings.hpp` with `Azure::Core::Strings::LocaleInvariantCaseInsensitiveEqual` and `Azure::Core::Strings::ToLower`.
-- Added `OperationCanceledException`.
 - Added `GetPort()` to `Url`.
 - Added `TransportPolicyOptions`.
 - Added `TelemetryPolicyOptions`.
 - Added `DateFormat::ToIso8601String(TimeFractionFormat)`.
```

Comment:
We added a bunch of DateTime APIs. Any particular reason to highlight this one, specifically?
**If not, remove it** (btw it should be DateTime, right, not DateFormat?).

We already have
> - Added `DateTime` supporting dates since 1601.